{
  "id": "CVE-2023-26150",
  "sourceIdentifier": "report@snyk.io",
  "published": "2023-10-03T05:15:49.963",
  "lastModified": "2023-10-03T12:51:52.930",
  "vulnStatus": "Awaiting Analysis",
  "descriptions": [
    {
      "lang": "en",
      "value": "Versions of the package asyncua before 0.9.96 are vulnerable to Improper Authentication such that it is possible to access Address Space without encryption and authentication.\r\r**Note:**\r\rThis issue is a result of missing checks for services that require an active session."
    },
    {
      "lang": "es",
      "value": "Las versiones del paquete asyncua anteriores a la 0.9.96 son vulnerables a una autenticaci\u00f3n incorrecta, por lo que es posible acceder al espacio de direcciones sin cifrado ni autenticaci\u00f3n. **Nota:** Este problema se debe a que faltan comprobaciones de servicios que requieren una sesi\u00f3n activa."
    }
  ],
  "metrics": {
    "cvssMetricV31": [
      {
        "source": "report@snyk.io",
        "type": "Secondary",
        "cvssData": {
          "version": "3.1",
          "vectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:L/I:L/A:N",
          "attackVector": "NETWORK",
          "attackComplexity": "LOW",
          "privilegesRequired": "NONE",
          "userInteraction": "NONE",
          "scope": "UNCHANGED",
          "confidentialityImpact": "LOW",
          "integrityImpact": "LOW",
          "availabilityImpact": "NONE",
          "baseScore": 6.5,
          "baseSeverity": "MEDIUM"
        },
        "exploitabilityScore": 3.9,
        "impactScore": 2.5
      }
    ]
  },
  "references": [
    {
      "url": "https://gist.github.com/artfire52/84f7279a4119d6f90381ac49d7121121",
      "source": "report@snyk.io"
    },
    {
      "url": "https://github.com/FreeOpcUa/opcua-asyncio/commit/2be7ce80df05de8d6c6ae1ebce6fa2bb7147844a",
      "source": "report@snyk.io"
    },
    {
      "url": "https://github.com/FreeOpcUa/opcua-asyncio/commit/b4106dfd5037423c9d1810b48a97296b59cde513",
      "source": "report@snyk.io"
    },
    {
      "url": "https://github.com/FreeOpcUa/opcua-asyncio/issues/1014",
      "source": "report@snyk.io"
    },
    {
      "url": "https://github.com/FreeOpcUa/opcua-asyncio/pull/1015",
      "source": "report@snyk.io"
    },
    {
      "url": "https://github.com/FreeOpcUa/opcua-asyncio/releases/tag/v0.9.96",
      "source": "report@snyk.io"
    },
    {
      "url": "https://security.snyk.io/vuln/SNYK-PYTHON-ASYNCUA-5673435",
      "source": "report@snyk.io"
    }
  ]
}